src/main/java/org/jabref/cli/JabRefCLI.java

```diff
@@ -288,6 +296,14 @@ private static Options getOptions() {
                 .argName("CITEKEY1[,CITEKEY2][,CITEKEYn] | PDF1[,PDF2][,PDFn] | all")
                 .build());
 
+        options.addOption(Option
+                .builder("j")
+                .longOpt("jumpToEntryKey")
```

Comment:
Rename to `jumpToKey`. (We use `CitationKey` throughout JabRef. Sometimes also `BibtexKey`. Never `EntryKey`. With `key` you are on the safe side ^^)